plutus-benchmark/marlowe/test/data.budgets.golden.tsv

```diff
@@ -0,0 +1,200 @@
+320693635	1461487	13652
```

Comment:
Can you open an issue for combining this with the tsv for the existing marlowe scripts, so we can easily compare them.

Reply:
Done: https://github.com/IntersectMBO/plutus-private/issues/1468